pkg/server/sandbox_run.go

```diff
@@ -126,7 +126,7 @@ func (c *criService) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 		// In this case however caching the IP will add a subtle performance enhancement by avoiding
 		// calls to network namespace of the pod to query the IP of the veth interface on every
 		// SandboxStatus request.
-		sandbox.IP, sandbox.CNIResult, err = c.setupPod(id, sandbox.NetNSPath, config)
+		sandbox.IP, sandbox.CNIResult, sandbox.CNIConfig, err = c.setupPod(id, sandbox.NetNSPath, config)
```

Comment:
CNIConfig is a daemon level thing. We should include it in `Status` instead of associate it with each pod.

I'm fine with cni result here, because it is per pod.

Reply:
See https://github.com/containerd/go-cni/pull/33#issuecomment-447471053